### DISCUSSION TITLE
Unreachable, invisible projects -- how to get them back visible?

### DISCUSSION THREAD

**Discussion by Keks-Dose** [community/community#138124] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

All of a sudden this project https://github.com/komascript/ vanished. I know the author and contacted him. He has no idea, why github wrote "Password reset notification due to suspicious activity", because he uses 2fa. After reset he can see his project, but nobody else, and this hasn't changed for weeks!

There are more projects bitten by this: https://github.com/orgs/community/discussions/135697 

Any ideas how to deal with this? Is there a way to complain here: https://support.github.com/request ??

**Comment by Shriharsh-Deshmukh** [comment]
If you're dealing with a vanished repository issue after a GitHub security alert, here's a step-by-step approach to resolve it:

1. **Check Repository Visibility:**
   - The repository may have been automatically set to private during the security incident. The owner should verify this under **Settings** > **General** > **Danger Zone** and ensure the repo is marked as public.

2. **Review 2FA and Security Logs:**
   - Double-check that 2FA settings are intact and secure. The owner can review the security logs in **Settings** > **Security** to spot any unusual activity that may have triggered the alert.

3. **Use Forks as a Temporary Solution:**
   - If the repository remains inaccessible, collaborators who have previously cloned the repo can push it to a new public repository as a temporary fix.

4. **Escalate to GitHub Support:**
   - If the issue persists, submit a support ticket through the [GitHub support form](https://support.github.com/request). Include details like the repository URL, what happened during the 2FA reset, and why the visibility issue persists.

5. **Monitor GitHub Discussions:**
   - Keep an eye on discussions like [this one](https://github.com/orgs/community/discussions/135697) for updates and similar cases, as others may have found solutions or received responses from GitHub.

This should help restore visibility to the affected repository.

**Comment by Keks-Dose** [comment]
> Escalate to GitHub Support:

    If the issue persists, submit a support ticket through the [GitHub support form](https://support.github.com/request). Include details like the repository URL, what happened during the 2FA reset, and why the visibility issue persists.

Thank you! I had a look at the site you mentioned, but the trouble is to find the correct topic, "Choose a topic to help us route your request quickly." Can you suggest under which topic an invisible account may fall? None of the possibilities seems to apply.

**Reply by komascript** [reply]
IMHO it is the first one: „Kontoberufung und -wiedereinsetzung” (n German, because I don't see the English page only the very partly translated German one). I've done all the steps, but the third one, because to get a visible fork, I would need a new account and the new account would need to access the repositories of the old flagged account, but the repositories of the old flagged account are invisible because they are owned by a flagged account.

**Reply by Shriharsh-Deshmukh** [reply]
For your issue regarding an invisible repository, you can choose the following topic in the GitHub support request form:

1. **Topic:** "Account Issues" or "Repository Management"
   - These categories generally cover issues related to account visibility, repository access, and related security concerns.

2. **Additional Step:** In the description box, clarify that the issue pertains to a repository becoming invisible despite correct settings after a security-related password reset. 

If there's no exact match, **"Other"** is a safe option, and you can detail the issue clearly in your request. This should ensure your request is routed correctly.

**Comment by komascript** [comment]
> and this hasn't changed for weeks!

Misunderstanding: My account has only been blocked for a few days. However, others have been affected by similar problems for weeks. And although I have explicitly reactivated my security keys and the Gitup app, the status is unfortunately not changing promptly. And unfortunately all my replies to discussions like this one are also invisible for others. So writing this is more or less useless …

**Comment by Keks-Dose** [comment]
Thank you, I opened a ticket and will report once the issue is solved.

**Comment by komascript** [comment]
The flag is removed and @komascript should be visible again.

**Comment by 0x676e67** [comment]
I still can't get a solution
